src/main/kotlin/com/github/pemistahl/lingua/internal/Constant.kt

```diff
@@ -16,9 +16,16 @@
 
 package com.github.pemistahl.lingua.internal
 
+import com.github.pemistahl.lingua.api.Language
+
 internal object Constant {
 
     val PUNCTUATION = Regex("\\p{P}")
     val NUMBERS = Regex("\\p{N}")
     val MULTIPLE_WHITESPACE = Regex("\\s+")
+
+    /**
+     * To define the languages  word split by NO SPACE , just like CHINESE, JAPANESE, KOREAN and so on.
+     */
+    val LANGUAGES_SPLIT_BY_NO_SPACE = listOf(Language.CHINESE, Language.JAPANESE, Language.KOREAN)
```

Comment:
1. Please remove the comment. It doesn't add any new information.

2. I'd rather call this constant `LANGUAGES_SUPPORTING_LOGOGRAMS`. A [logogram](https://en.wikipedia.org/wiki/Logogram) is the correct term for a single character that represents an entire word.

Reply:
Thanks for you suggestion.  The item of logogram  will be much better than languages. And I will remove the comment, becasue the meaning can be recognized by variable name.